qa/tasks/cephfs/test_nfs.py

```diff
@@ -118,3 +169,26 @@ def test_create_multiple_exports(self):
         self._create_export(export_id='4', extra_cmd=[self.pseudo_path+'3', fs_path.strip()])
         self._test_delete_cluster()
         self._check_export_obj_deleted(conf_obj=True)
+
+    def test_exports_on_mgr_restart(self):
+        self._create_default_export()
+        self._unload_module("cephadm")
+        self._load_module("cephadm")
+        self._orch_cmd("set", "backend", "cephadm")
+        self._test_list_export()
+        self._delete_export()
+
+    def test_exports_create_with_invalid_fsname(self):
+        try:
+            self._test_create_cluster()
+            self._create_export(export_id='1')
+        except CommandFailedError:
+            # Command should fail for test to pass
+            pass
```

Comment:
If the `try:` block should succeed, then you should fail the test?

Reply:
Done